cryptoki/src/mechanism/mechanism_info.rs

```diff
@@ -228,6 +231,27 @@ impl MechanismInfo {
     pub fn ec_compressed(&self) -> bool {
         self.flags.contains(MechanismInfoFlags::EC_COMPRESS)
     }
+
+    /// True if the mechanism can be used to encrypt messages
+    ///
+    /// See [`Session::encrypt_message`](crate::session::Session::encrypt_message)
```

Comment:
I wonder why this line couldn't just be:

```suggestion
    /// See [`Session::encrypt_message`].
```

Is it because `Session` is not in scope? (then it can be imported above with `#[cfg(doc)]` attribute) (we do something similar [in signstar](https://gitlab.archlinux.org/archlinux/signstar/-/merge_requests/202/diffs#07c064b304098b8d77cafe580429e647de21dc76_12_11))

Reply:
I just copy-pasted what other flags have to make it consistent. We can do something you proposed in separate issue or PR if it can be simplified, though, as it would be touching more parts not related to the changes here.